rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
```

Comment:
-  [ ] The rental should add itself to the @book achieving the 'belongs-to' association this is how it would look like like 👇 
```ruby
book.rentals << self
```